bundles/org.openhab.ui/web/src/assets/i18n/setup-wizard/en.json

```diff
@@ -21,6 +21,7 @@
   "setupwizard.addons.title": "Install Add-ons",
   "setupwizard.addons.header1": "Most of openHAB's functionality is provided by add-ons.",
   "setupwizard.addons.header2": "Choose which ones you'd like to install right away.",
+  "setupwizard.addons.header3": "Add-ons recommended for functional reasons are pre-selected.",
```

Comment:
@rkoshak What do you think about the wording?

Reply:
As written it seems mysterious and raises questions. What functional reasons? Maybe something like

> We've pre-selected some add-ons to provide a fully capable environment to start with.

I'm not really happy with that wording either but it at least heads off the questions raised.

Reply:
I would keep the language plain, how about:
```
"setupwizard.addons.title": "Install Add-ons",
"setupwizard.addons.header1": "Add-ons extend openHAB's functionality.",
"setupwizard.addons.header2": "Choose which add-ons you would like to install now.",
"setupwizard.addons.header3": "A recommended set of add-ons have been pre-selected.",
```

Reply:
Even better. I like that wording the best!

Reply:
Okay, thanks for your input.
Much better than what I came up with!

I’ll change it.

Reply:
Done, I have changed 2 and 3 and left 1 as is. I prefer the current wording for 1.